### ISSUE TITLE
Problems in Ubuntu 16.10

### ISSUE THREAD

**Issue by lenalebt** [m0sia/pyParrotZik#17]
I have problems running the `parrot_zik_tray` tool under Ubuntu 16.10. It just does not find my Parrot Zik 2.0. Bluetooth audio output works on the headphones, as well does general input (pause/unpasue, next/previous track, volume changes), it is just that the tray tool does not find the headphones.

Under Ubuntu 16.04, everything works as expected.

I am unsure what info you need in addition to make an educated guess what might be wrong. I'd be willing to try to make changes in the code as well, but I am not on the pythonic side of the force

**Comment by brisssou** [comment]
somehow, I got this working, but it really feels like hacking.

### TL;DR:
running `bluetoothctl` gives you the mac for your Zik.

You can then edit `parrot_zik/bluetooth_paired_devices.py` with 

    sed -i 's|^    mac = |    mac = <your_actual_mac>  # |' parrot_zik/bluetooth_paired_devices.py


and run `sudo -H python setup.py install` again.

### What led me there:
I was facing the same issue you did, I tried running pyParrotZik in debug in my IDE. I turned out working. But when I run it, it seems like the fallback to `bluetoothctl` somehow fails to guess the mac address.
I'm trying to get this to work.
I also had to copy the icons to `/usr/share/icons/` to have them to actually show up.

**Comment by lenalebt** [comment]
Works for me, although it is not ideal. Thanks for your input!

**Comment by lenalebt** [comment]
Just for others as a note: Working on Ubuntu 18.04 without these problems.
